data/tools/wesnoth/wmltools3.py

```diff
@@ -36,6 +48,14 @@ def __init__(self, sub, start, end):
     def __repr__(self):
         return "<Class Substitution, sub={}, start={:d}, end={:d}>".format(self.sub, self.start, self.end)
 
+def is_map(filename):
+    "Is this file a map?"
+    for ext in map_extensions:
```

Comment:
if you assume extension does not contain dot, function can be simplified

Reply:
You mean `filename.endswith(ext)`? I don't want to accidentally match something like `testmap`.

Reply:
endswith "."+ext

Reply:
but I was thinking of `filename.split('.')[-1] in map_extensions`

Reply:
> but I was thinking of `filename.split('.')[-1] in map_extensions`

Great idea, I've implemented it.